### PR TITLE
Symfony BC

### PR DESCRIPTION
**Pull request by kyrylich**

### CODE REVIEW

File: src/Resources/views/Collector/cassandra.html.twig [hunk 1]
```diff
@@ -16,9 +16,9 @@
 
 {% block menu %}
     <span class="label">
-    <span class="icon">{% spaceless %}
+    <span class="icon">{% apply spaceless %}
```
Comment (Oliboy50) [inline src/Resources/views/Collector/cassandra.html.twig]:
```suggestion
    <span class="icon"><img ... /></span>
```

using `apply spaceless` is a BC break for those using an old Twig version, so we'd prefer to simply remove the need for `spaceless` by simply removing the spaces "manually"

File: src/Resources/views/Collector/cassandra.html.twig [hunk 2]
```diff
@@ -16,9 +16,9 @@
...
     <img alt="Cassandra" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABwAAAAcCAYAAAByDd+UAAACAElEQVRIie3VS4hOYRzH8c+8YkjGYDAbFpoZJJcJC6XYzM6lLCTZka2NsnBJKZKQS1IWKErJwqWk3FbCwmVBCSmSxkKjRCSvxfmfnDnzvued90WzmV+dnvP8L8/36f+c/3MY1hBpLDbhBM5gNxb/D9BmvMIVrENHwFswDz1Yhgl/C2rHGzzCxEHEd2JaZj45Ny/UDJRxso4NZkHwMNaoqeYIvNAADGbGeC8DHKOgSrfxsoK9hEO4LynzNSzJxfQE5B364v1ZjOcqweaHc1XOvhw/wvcWz+O9jFOZuDZswd2M/wX2Ym4l4OPYWVatkfgJ3Tnf+fBtz9n3hP0XnlQCQVPAjubsaUlaquRdD39rzNOy3sB6A6vQT2VsyG3iK/ZVS8BozMbImI/CQcmZwwFVPpgpBp7fiLCtLgAWqVTkbI/F12RsTWFb2yBwXJEzXXxrztaHiwV5XXgdY17TawE/407Ofik2srJKXtpvzRV8Ne/XtL/yge/xE6clZ12SlPlDxOfbheT8x9cCLo0Fzubsk3DLn2ZOny8GXhKpttWCpXoQi1W6GbqwA/v1/7jyWihpj0GpLYDfVG/2InVIrrK6tCig37GgjrwVOF4vLFWn5IzKuIk5BbHdeIqNjcKy2oneAH+U9ORhHJH8onpxGVP/BSyrWdgl6curOCYp4bCGXr8BrHF1XeEAlBEAAAAASUVORK5CYII=" />
-        {% endspaceless %}</span>
+        {% endapply %}</span>
```
Comment (Oliboy50) [inline src/Resources/views/Collector/cassandra.html.twig]:
```suggestion 
<span class="icon"><img ... /></span>
```

File: src/Resources/views/Collector/cassandra.html.twig (hunk 1 above)
Comment (kyrylich) [reply src/Resources/views/Collector/cassandra.html.twig]: @Oliboy50 I have commited your suggestion. Already is all right ?

**Comment by Oliboy50** [comment]
ℹ️ the CI build is failing for `master` too, because of: 
> E: Failed to fetch http://ppa.launchpad.net/linuxjedi/ppa/ubuntu/dists/xenial/main/binary-amd64/Packages  404  Not Found

=> this PR build: https://travis-ci.org/M6Web/CassandraBundle/builds/659191384
=> the current `master` build: https://travis-ci.org/M6Web/CassandraBundle/builds/659219117

so I think we should merge this PR as is and fix the CI build in another PR

@b-viguier what do you think?

**Comment by b-viguier** [comment]
@Oliboy50 Sure, but would be better to fix the CI **before** to merge the CI 😅

**Comment by kyrylich** [comment]
Hi everyone, any news about this PR? 
Should I fix something ?

**Comment by Oliboy50** [comment]
@kyrylich @b-viguier I've opened a PR to fix the current Travis build
=> https://github.com/M6Web/CassandraBundle/pull/45

**Comment by Oliboy50** [comment]
this is now part of the [3.0.0 release](https://github.com/M6Web/CassandraBundle/releases/tag/v3.0.0)

Thank you for your time

**Comment by kyrylich** [comment]
Thanks a lot!

**Pull request merged**